include/openenclave/corelibc/bits/addrinfo.h

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+struct __OE_ADDRINFO
+{
```

Comment:
For basic posix types like addrinfo  and oe_socklen_t , given we are enabling POSIX API, I am wondering what value it has by defining exact one to one OE_* matching type. I noticed direct castings are used when calling actual host side socket APIs.

Reply:
Good question. We provide the oe-prefixed types so they can be passed to the host side in OCALLs. Without the oe-prefixed names, these would conflict with Windows definitions. For example, many of these integer typedefs have different sizes on Windows.

Reply:
Thanks for addressing my comment.